README.md

```diff
@@ -32,6 +40,21 @@ QUICKINSTALL:
 
 After building go to the `examples` directory and have a look at the python scripts.
 
+DOCUMENTATION:
+==============
+
+Documentation for the developer version of ESPResSo++ is at:
+
+http://espressopp.github.io
+
+Documentation for release versions from v1.9.4.1 onward is at:
```

Comment:
I put v1.9.4.1 as the example because it's the only alternative doc version at the moment, right? But in the future when presumably we will have several releases and corresponding docs we can put something else.